service/topology_coordinator.cc

```diff
@@ -2021,6 +2021,8 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
                             throw;
                         } catch (group0_concurrent_modification&) {
                             throw;
+                        } catch (raft::request_aborted&) {
```

Comment:
We need to make sure that none of those can be a result of a timeout.

Reply:
Looking at `raft_server_with_timeouts::run_with_timeout`, I think we are sure it cannot be a result of a timeout. There is an edge case when we throw `raft::request_aborted` for a timeout during abort, but it seems reasonable.